c2corg_ui/static/js/map/map.js

```diff
@@ -887,5 +889,16 @@ app.MapController.prototype.resetFeature = function() {
   this.scope_.$root.$emit('mapFeaturesChange', features, true /* isReset */);
 };
 
+/**
+ * @private
+ */
+app.MapController.prototype.updateMapResize_ = function() {
+  setTimeout(function() {
```

Comment:
I initialized `ngeo.debounce` and wrote 
```
  this.ngeoDebounce_(function() {
    this.map.renderSync();
    this.map.updateSize();
  }.bind(this), 300, true);
```

but the functions are not even being called. Any ideas?